### PR TITLE
Behat speaker can edit talk

### PR DESCRIPTION
**Pull request by leovujanic**
Added behat - speaker can edit talk.
Implemented related methods on TalkEntity

### CODE REVIEW

File: features/edit-talk.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@talk
+Feature:
+  As a speaker
+  I shoould be able to edit my talk
```
Comment (HcsOmot) [inline features/edit-talk.feature]: Please fix typo

File: features/edit-talk.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  Scenario: Speaker has rights to edit talk title
+    Given Given "Alex Smith" is a speaker on the talk "Alexis talk"
```
Comment (HcsOmot) [inline features/edit-talk.feature]: This second 'Given' isn't needed.

File: src/Talk/Behat/TalkDomainContext.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+class TalkDomainContext implements Context
+{
+    /**
+     * @var TalkEntity
+     */
+    private $talk;
+
+    /**
+     * @Given Given :userName is a speaker on the talk :talkTitle
```
Comment (HcsOmot) [inline src/Talk/Behat/TalkDomainContext.php]: This second 'Given' isn't needed

File: features/edit-talk.feature [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    When speaker change talk title to "New Alexis talk"
```
Comment (HcsOmot) [inline features/edit-talk.feature]: Please fix grammar, 'change' -> 'changes'

File: features/edit-talk.feature [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    Then the talk title is "New Alexis talk"
+
+  Scenario: Speaker has rights to edit talk description
+    Given "Alex Smith" is a speaker on a "Alexis talk"
+    When speaker change talk description to "New Description for Talk"
```
Comment (HcsOmot) [inline features/edit-talk.feature]: same as above

**Comment by msvrtan** [comment]
I'm closing this down as nothing has been done on this PR for a really long time :(

**Pull request closed without merging**